students/gyuchan/users/views.py

```diff
@@ -33,5 +32,26 @@ def post(self, request):
                 phone_number = user_phone_number
             )
             return JsonResponse({'message':'CREATED'}, status=201)
+        except KeyError:
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+
+class LogInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            
+            user_id       = data['id']
+            user_password = data['password']
+            
+            user_table = User.objects.filter(email=user_id)
```

Comment:
```suggestion
            user_table = User.objects.filter(email=user_id)
```
Django의 QuerySet AP에는 하나의 객체만 가져오는 메소드가 존재합니다. 굳이 filter()로 가져올 필요가 없어 보입니다.
추가로 user_table이라는 변수명은 조금 어색해보이네요.